apis/reaper/v1alpha1/reaper_types.go

```diff
@@ -56,7 +56,7 @@ type ReaperTemplate struct {
 
 	// Defines the secret which contains the username and password for the Reaper UI and REST API authentication.
 	// +optional
-	UiUserSecretRef corev1.LocalObjectReference `json:"uiUserSecretRef,omitempty"`
+	UiUserSecretRef *corev1.LocalObjectReference `json:"uiUserSecretRef,omitempty"`
```

Comment:
**Todo:** Could you add a comment here explaining how to disable auth? That'll end up in the CRD docs.

Reply:
Sure.